frontend/src/views/analysis_view.js

```diff
@@ -567,6 +568,49 @@ export class AnalysisView extends React.Component {
                                 />
                             </Tab>
                             <Tab eventKey="Relevant Words" title="Relevant Words">
+                                <TabularAnalysis
+                                    title = {
+                                        "Percentage and Frequency of Relevant Words per Question"
+                                    }
+                                    subtitle = {
```

Comment:
This should be factored out and provided as data from the backend, rather than hardcoded into the interface.